cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -187,4 +187,17 @@ describe('Top toolbar tests.', function() {
 		cy.get('#copy-paste-container table')
 			.should('exist');
 	});
+
+	it('Insert image.', function() {
+		cy.get('#tb_editbar_item_insertgraphic')
+			.click({force: true});
+
+		cy.get('#insertgraphic[type=file]')
+			.attachFile('/mobile/writer/image_to_insert.png');
```

Comment:
Here we should not use an image file used by mobile tests. The data files are under cypress_test/data folder. You should copy the image file from the mobile folder to the desktop/writer/ folder and use that here. In this way, we can keep the mobile tests and the desktop tests independent from each other.